interfaces/builtin/network_control.go

```diff
@@ -298,8 +298,9 @@ umount /,
 capability sys_ptrace,
 
 # 'ip netns exec foo /bin/sh'
-mount options=(rw, rslave) /,
+#mount options=(rw, rslave) /, # commented out because of LP: #2023025
```

Comment:
this will fix the compile failure in LP: #2023025. It would lead to a tightening of policy except the rule on line 302 is wider, which means as long as that rule is part of policy this rule is not needed.